packages/live-preview-sdk/src/index.ts

```diff
@@ -249,12 +266,28 @@ export class ContentfulLivePreview {
     return this.liveUpdatesEnabled;
   }
 
-  static openEntryInEditor({ fieldId, entryId, locale }: LivePreviewProps): void {
-    if (!fieldId || !entryId) {
-      debug.error('Please provide field id and entry id to openEntryInEditor.');
+  static openEntryInEditor(props: LivePreviewProps): void {
```

Comment:
should be open entity in Editor instead

Reply:
Will be breaking change, thus leaving as it is for now.